test/acceptance/features/steps/service_binding.py

```diff
@@ -86,6 +86,8 @@ def invalid_sbr_is_applied(context):
 def sbo_is_ready(context, sbr_name=None):
     if sbr_name is None:
         sbr_name = list(context.bindings.values())[0].name
+    else:
+        sbr_name = Template(sbr_name).substitute(scenario_id=scenario_id(context))
```

Comment:
not seeing its usage somewhere in this PR?

Reply:
not used, now - removing as not related to the PR